src/main/java/com/coreos/jetcd/EtcdWatchImpl.java

```diff
@@ -0,0 +1,417 @@
+package com.coreos.jetcd;
+
+import com.coreos.jetcd.api.*;
+import com.coreos.jetcd.data.ByteSequence;
+import com.coreos.jetcd.options.WatchOption;
+import com.coreos.jetcd.watch.WatchCreateException;
+import com.google.protobuf.ByteString;
+import io.grpc.ManagedChannel;
+import io.grpc.stub.StreamObserver;
+import javafx.util.Pair;
```

Comment:
You should not include `javafx` as a dependency< nothing else in this project does. It also means this only works with Oracle JDKs (i.e. it won't work with OpenJDK).  If need be, add a package-local class at the end of the file like this:

````
class Pair<K,V> {
    final K k;
    final V v;
    Pair(K k, V v) { this.k = k; this.v = v; }
    K getKey() { return k; }
    V getValue() { return v; }

    @Override
    public int hashCode() {
        return k.hashCode() + v.hashCode();
    }

    @Override
    public boolean equals(Object obj) {
        if (obj==null || !(obj instanceof Pair)) return false;
        Pair<K,V> other = (Pair<K,V>)obj;
        return k.equals(other.k) && v.equals(other.v);
    }
}
````